spec/functional/resource/env_spec.rb

```diff
@@ -55,6 +79,25 @@
         expect(ENV[chef_env_test_lower_case]).to eq(env_value1)
       end
 
+      it "should create an environment variable with default user System for action create" do
+        expect(ENV[chef_env_test_lower_case]).to eq(nil)
+        test_resource.key_name(chef_env_test_lower_case)
+        test_resource.value(env_value1)
+        test_resource.run_action(:create)
+        env_obj
+        expect(@env_obj.username.upcase).to eq(default_env_user)
```

Comment:
Can we assign the return value of `env_obj` to some variable instead of using the instance variable `@env_obj`?